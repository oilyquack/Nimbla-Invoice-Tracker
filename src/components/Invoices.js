import React from "react";
import Invoice from "./Invoice";

class Invoices extends React.Component {
  constructor() {
    super();
  }

  render() {
    return (
      <div>
        {this.props.invoices.map(invoice => {
          return (
            <Invoice
              invoiceId={invoice.id}
              amount={invoice.amount}
              payBy={invoice.payBy}
              customer={invoice.customer}
              services={invoice.services}
            />
          );
        })}
      </div>
    );
  }
}

export default Invoices;
