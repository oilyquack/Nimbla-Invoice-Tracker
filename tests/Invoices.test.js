import React from "react";
import renderer from "react-test-renderer";
import Invoices from "../src/components/Invoices";

it("matches the snapshot", () => {
  const tree = renderer.create(<Invoices />);
  expect(tree).toMatchSnapshot();
});
