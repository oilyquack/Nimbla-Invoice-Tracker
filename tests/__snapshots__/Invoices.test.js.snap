// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="app__invoices"
>
  <select
    onChange={[Function]}
    value=""
  >
    <option
      value="all"
    >
      All
    </option>
    <option
      defaultValue={true}
      value="paid"
    >
      Paid
    </option>
    <option
      value="unpaid"
    >
      Unpaid
    </option>
  </select>
  <div
    className="app__invoices__display"
  />
</div>
`;
